### PR TITLE
fixied flash of magnifying icon

### PR DESCRIPTION
**Pull request by darushka**
This PR fixes flashing issue with Magnifying Glass icon when closing search.

### CODE REVIEW

File: src/directives/search/search.controller.js [hunk 1]
```diff
@@ -9,8 +9,11 @@ class SearchController {
   }
 
   toggle(sidenavId) {
-    this.isOpen = !this.$mdSidenav(sidenavId).isOpen();
-    return this.$mdSidenav(sidenavId).toggle();
+    let _this = this;
```
Comment (jsmylnycky) [inline src/directives/search/search.controller.js]: Can you please remove the "let _this = this" syntax?

**Comment by jsmylnycky** [comment]
:+1:  once the _this syntax is cleaned up

**Comment by darushka** [comment]
@jsmylnycky _this removed

**Comment by darushka** [comment]
Merged squash https://github.com/peachworks/ngPeach/commit/df0708be749c780dea0adb6a8842b2483eec547f

**Comment by jsmylnycky** [comment]
v0.4.1

**Pull request closed without merging**